iphone/Classes/TiUIListView.m

```diff
@@ -2011,6 +2012,15 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView
         TiUIListSectionProxy *section;
         CGFloat topSpacing = scrollView.contentOffset.y + scrollView.adjustedContentInset.top;
 
+        NSString *direction = nil;
```

Comment:
```suggestion
        NSString *direction = [NSNull null];
```

Otherwise, if the `if` and `else if` are both not matching, it would remain `nil` and crash the app, as `nil` cannot be bridged to a JS type. Therefore, `nil` always should default to `[NSNull null]` which is the equivalent to `null` in JS.

Reply:
I see. Should I change it here too: https://github.com/tidev/titanium-sdk/blob/a7ac1d42103859289e61806b24727fc6844e5554/iphone/Classes/TiUIListView.m#L2125 I've copied it from there

Reply:
Ideally yes